contracts/hardhat.config.ts

```diff
@@ -70,6 +72,14 @@ export const networks: {[index: string]: NetworkUserConfig} = {
     url: `${apiUrls.baseGoerli}`,
     gasPrice: ethers.utils.parseUnits('0.0000001', 'gwei').toNumber(),
   },
+  abitrum: {
```

Comment:
There is a typo here

Reply:
Fixed in af0be22f5a302bdb5ca62aa59691cd7abf8ebf52